### PR TITLE
chore: Count sources by their required input vname corpus

### PR DESCRIPTION
**Pull request by salguarnieri**

### CODE REVIEW

File: kythe/proto/analysis.proto [hunk 1]
```diff
@@ -188,25 +188,27 @@ message KzipInfo {
...
-      // Number of required inputs mentioned in these compilation units.
-      int32 required_inputs = 3;
     }
 
     // Map from compilation unit language to stats about compilation units with
     // that language.
     map<string, CompilationUnits> language_compilation_units = 3;
 
     message RequiredInputs {
+      // TODO(salguarnieri) Add count of unique inputs for this corpus?
```
Comment (justbuchanan) [inline kythe/proto/analysis.proto]: If we de-duplicate the inputs to get a unique count, is there reason to keep the old counter around too?

If we only need the one number, could we replace the `RequiredInputs` messages with just an int?

File: kythe/proto/analysis.proto [hunk 2]
```diff
@@ -188,25 +188,27 @@ message KzipInfo {
     // The count of data present in compilation units for a given corpus and
     // language.
     message CompilationUnits {
```
Comment (justbuchanan) [inline kythe/proto/analysis.proto]: Do we need this wrapper message anymore?

Comment (salguarnieri) [reply kythe/proto/analysis.proto]: I'd like to keep it so if we decide we want to record more info about the CU in the future we don't have to make a new map again

Comment (salguarnieri) [reply kythe/proto/analysis.proto]: But if you prefer I can get rid of it

Comment (justbuchanan) [reply kythe/proto/analysis.proto]: :+1:

**Pull request merged**